File: read_pdf.js
const fs = require('fs');
const pdf = require('pdf-parse');

let dataBuffer = fs.readFileSync('./Manual_Integracio_n_API_2_3.pdf');

pdf(dataBuffer).then(function(data) {
  fs.writeFileSync('getnet_manual_extracted.txt', data.text);
  console.log('PDF extracted successfully. Look at getnet_manual_extracted.txt');
}).catch(err => {
  console.error("Error parsing PDF:", err);
});

File: read_pdf.py
import fitz  # PyMuPDF
try:
    doc = fitz.open("Manual_Integracio_n_API_2_3.pdf")
    text = ""
    for page in doc:
        text += page.get_text()
    with open("getnet_manual_extracted.txt", "w", encoding="utf-8") as f:
        f.write(text)
    print("Success")
except Exception as e:
    print(f"Failed: {e}")
